src/components/CompanyFilterButton/index.tsx

```diff
@@ -85,15 +130,56 @@ function TooltipContent(props: CompanyFilterButtonUIProps): ReactElement {
     );
 }
 
+function TooltipContent(props: CompanyFilterButtonUIProps): ReactElement {
+    const { hideTooltip, companiesQuery, selectedIndex, selectIndex, onChange, inputRef, setState } = props;
+
+    // Need to be able to hide the tooltip on keypress.
+    useElementListener(
+        'keydown',
+        (rawEvent: Event) => {
```

Comment:
I think with the new event listener stuff you shouldn't have to do this `rawEvent` and cast thing anymore. Let me know if it gives you trouble still though.